testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -433,6 +446,7 @@ Feature: Synchronize products in the products page of the Setup Wizard
 @sle15sp5s390_minion
   Scenario: Add SUSE Linux Enterprise Server 15 SP5 for s390x Uyuni Client tools
     When I use spacewalk-common-channel to add channel "sles15-sp5-devel-uyuni-client" with arch "s390x"
+    And I wait until all synchronized channels for "sles1-sp5-s390x" have finished
```

Comment:
```suggestion
    And I wait until the channel "sles15-sp5-devel-uyuni-client" has been synced
```


But.... I'm not sure how that will work with multiple architectures.

Reply:
the code seems to be using:
```
dumpsolv /var/cache/rhn/repodata/#{channel}/solv
zcat /var/cache/rhn/repodata/#{channel}/*primary.xml.gz
```
That looks like it's architecture-independent...